gwcs/converters/wcs.py

```diff
@@ -0,0 +1,210 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+# -*- coding: utf-8 -*-
+
+from asdf.extension import Converter
+
+
+_REQUIRES = ['astropy']
+
+
+__all__ = ["WCSConverter", "CelestialFrameConverter", "CompositeFrameConverter",
+           "FrameConverter", "SpectralFrameConverter", "StepConverter",
+           "TemporalFrameConverter", "StokesFrameConverter"]
+
+
+class WCSConverter(Converter):
+    tags = ["tag:stsci.edu:gwcs/wcs-*"]
+    types = ["gwcs.wcs.WCS"]
+
+    def from_yaml_tree(self, node, tag, ctx):
+        from ..wcs import WCS
+        name = node['name']
+        steps = [(x.frame, x.transform) for x in node['steps']]
```

Comment:
Quite possibly it might work, but that is how the existing tag is implemented:

https://github.com/spacetelescope/gwcs/blob/464e55ff1dc2a27dc2329cd43043071bf2cd5165/gwcs/tags/wcs.py#L27-L30

I do not know if your suggestion might have any side-effects. Let's wait for @nden to comment on this.

Reply:
I think it is fine to simplify the code.